src/main/java/com/crud/tasks/trello/client/TrelloClient.java

```diff
@@ -66,13 +56,14 @@ public CreatedTrelloCard createNewCard(TrelloCardDto trelloCardDto){
         return restTemplate.postForObject(url,null,CreatedTrelloCard.class);
     }
 
-    private URI buildUrl(){
+    /*private URI buildUrl(){
```

Comment:
jeśli masz jakieś niepotrzebne części kodu to je usuwaj. strasznie źle się czyta kod w którym jest sporo nieużywanych pokomentowanych rzeczy